lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2444,6 +2488,36 @@ def _get_sig_info(
         assert isinstance(sig_info, GalleryImageVersion), f"actual: {type(sig_info)}"
         return sig_info
 
+    def _get_cgi_info(
+        self, community_gallery_image: CommunityGalleryImageSchema
+    ) -> CommunityGalleryImageVersion:
+        compute_client = get_compute_client(self)
+        cgi_info = compute_client.community_gallery_image_versions.get(
+            location=community_gallery_image.location,
+            public_gallery_name=community_gallery_image.image_gallery,
+            gallery_image_name=community_gallery_image.image_definition,
+            gallery_image_version_name=community_gallery_image.image_version,
+        )
+        assert isinstance(
+            cgi_info, CommunityGalleryImageVersion
+        ), f"actual: {type(cgi_info)}"
+        return cgi_info
+
+    @lru_cache(maxsize=10)  # noqa: B019
+    def _get_detailed_cgi(
```

Comment:
Is the word "detailed" from the document? Use the accurate word to get it clear.

Reply:
updated